cmd/ackdev/cmd/add_repo.go

```diff
@@ -36,33 +42,38 @@ var addRepositoryCmd = &cobra.Command{
 }
 
 func addRepository(cmd *cobra.Command, args []string) error {
-	for _, service := range args {
-		service = strings.ToLower(args[0])
+	cfg, err := config.Load(ackConfigPath)
+	if err != nil {
+		return err
+	}
 
-		ctx := cmd.Context()
+	repoManager, err := repository.NewManager(cfg)
+	if err != nil {
+		return err
+	}
 
-		cfg, err := config.Load(ackConfigPath)
-		if err != nil {
-			return err
-		}
+	for _, service := range args {
+		service = strings.ToLower(args[0])
```

Comment:
Oops! This is a bug I introduced in the last PR. Would you mind changing it?
```suggestion
		service = strings.ToLower(service)
```